services/task-svc/migrations/000004_init.up.sql

```diff
@@ -0,0 +1,9 @@
+DROP TABLE beds;
```

Comment:
maybe don't drop the table every time we migrate :D

Reply:
I think it's because we already created that table in the previous migrations.

Reply:
then we should `alter if exists` it or not?

Reply:
😅